### PR TITLE
Graphql/resolvers

### PR DESCRIPTION
**Pull request by fbielejec**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -6,7 +6,7 @@
                  ;; General
                  [akiroz.re-frame/storage "0.1.4"]
                  [camel-snake-kebab "0.4.0"]
-                 [cljs-web3-next "0.1.1"]
+                 [cljs-web3 "0.19.0-0-11"]
```
Comment (jpmonettas) [inline project.clj]: why are we going back to old cljs-web3?

File: project.clj [hunk 2]
```diff
@@ -152,10 +143,11 @@
               [lein-doo "0.1.10"]]
     :repl-options {:init-ns user
                    :nrepl-middleware [cider.piggieback/wrap-cljs-repl]}}}
+
   :cljsbuild
   {:builds
    [{:id "dev-ui"
-     :source-paths ["src" "test" "dev/ui"]
+     :source-paths ["src/ethlance/ui" "src/ethlance/shared" "dev/ui"]
```
Comment (jpmonettas) [inline project.clj]: what is the reason for going back to this?

File: src/ethlance/server/db.cljs [hunk 3]
```diff
@@ -74,7 +74,7 @@
      ;; PK
      [(sql/call :primary-key :user/address)]
      ;; FKs
-     [[(sql/call :foreign-key :user/address) (sql/call :references :User :user/address)]]]
+     [(sql/call :foreign-key :user/address) (sql/call :references :User :user/address) (sql/raw "ON DELETE CASCADE")]]
```
Comment (jpmonettas) [inline src/ethlance/server/db.cljs]: adding all this "on delete cascade" looks to me like we are planning to delete records from our db. Do we have this requirements in ethlance? Or what is its purpose?

File: src/ethlance/shared/graphql/utils.cljs [hunk 4]
```diff
@@ -0,0 +1,59 @@
+(ns ethlance.shared.graphql.utils
```
Comment (jpmonettas) [inline src/ethlance/shared/graphql/utils.cljs]: Isn't this functionality the stuff provided by https://github.com/district0x/graphql-query/ ?

File: src/ethlance/server/db.cljs (hunk 3 above)
Comment (fbielejec) [reply src/ethlance/server/db.cljs]: it's for: https://github.com/district0x/ethlance/blob/graphql/resolvers/src/ethlance/server/db.cljs#L470

File: project.clj (hunk 2 above)
Comment (fbielejec) [reply project.clj]: fixed

File: project.clj (hunk 1 above)
Comment (fbielejec) [reply project.clj]: rebase gone wrong, fixed

File: src/ethlance/shared/graphql/utils.cljs (hunk 4 above)
Comment (fbielejec) [reply src/ethlance/shared/graphql/utils.cljs]: no, the other way around - this ns is using that lib.

**Pull request merged**